user-guide/Standard_Apps/EPM/EPM_VSAT/VSAT_components.md

```diff
@@ -0,0 +1,189 @@
+
+### PLM
+
+During planned maintenance (PLM), DataMiner suppresses any diagnostic and platform falls alerts related to the event process. It works in conjunction with SRA modules.
+
+## Correlation rules
+
+The VSAT EPM Solution uses the following Correlation rules:
+
+- **VerSatOnReportCorrelation**: Correlation rule in charge of the synchronization of RDS subscribers (KPI, Entry, DCAT, etc.) by monitoring for **OnChange* and *OnUpdate* information events.
+
+- **VrzVsatEventManager**: Correlation rule in charge of picking up all the events/alarms that should create a ticket request entry in the WM Ticketing protocol and send it to the correlation that send the requests to the proper ticketing element.
+
+## Automation scripts
+
+The VSAT EPM Solution uses the following Automation scripts:
+
+- **VerSatOnReportAutomation**: Automation script in charge of updating and triggering the logic to keep RDS tables updated and synchronized across all DMAs (*Config Entry Subscribers*, *Entity Subscribers*, *KPI Entry Subscribers*, and *DCAT Profiles*).
+
+  This Automation script is dependent on the following protocols:
+
+  - Verizon Reports and Dashboards Solution: This protocol exports and imports data related to different collector elements.
+
+  - Verizon WM RDS: This protocol manages the interaction between the Verizon Reports and Dashboards Solution protocol and the Profile Manager app. The information stored in the tables of the Verizon Reports and Dashboards Solution is stored in the Profile Manager app.
+
+- **VerSatOnSunOutageAutomation**: Automation script in charge of handling sun outage protocol events.
+
+  This Automation script is dependent on the following protocols:
+
+  - Verizon WM DSM: This system protocol allows the Verizon DSM SO protocol to communicate with the Profile Manager.
+
+  - Verizon DSM SO: This protocol is used to gather information via inter-element communication that will be exported to a location used by the Generic Sun Outage protocol. The information gathered consists of key parameters used during the calculation of sun outages. This protocol is purely a system protocol with this sole responsibility.
+
+  The *VerSatOnSunOutageAutomation* script oversees collecting data from an information event triggered by Verizon DSM SO and passes it to Verizon WM DSM in order for that workflow manager to create, update, or remove a Profile Manager object. The information event is triggered from the context menus on the DSM SO protocol.
+
+- **EPM BE Message Handler**: Automation script in charge of handling EPM back-end messages.
+
+  This Automation script is dependent on the following protocols:
+
+  - The [collector](#collector-connectors) that triggered the initial EPM Message handler script:
```

Comment:
Capitalization is inconsistent here and in line 103 and 115: Either we should fully capitalize "EPM Message Handler" script to refer to its exact name, or we should not capitalize this at all if it's intended as a description instead of the exact name.